doc/classes/Vector2.xml

```diff
@@ -264,12 +264,26 @@
 				Returns the vector with a maximum length by limiting its length to [param length].
 			</description>
 		</method>
+		<method name="max" qualifiers="const">
+			<return type="Vector2" />
+			<param index="0" name="with" type="Vector2" />
+			<description>
+				Returns the component-wise maximum of this and [param with], equivalent to [code]Vector2(max(x, with.x), max(y, with.y))[/code].
```

Comment:
```suggestion
				Returns the component-wise maximum of this and [param with], equivalent to [code]Vector2(maxf(x, with.x), maxf(y, with.y))[/code].
```
Have already updated this and will push once CI completes, same but with `maxi/mini` for the integer vectors